packages/xforms-engine/src/lib/reactivity/createItemCollection.ts

```diff
@@ -120,11 +123,30 @@ const createItemsetItems = (
 					const value = createComputedExpression(context, itemset.value, {
 						defaultValue: '',
 					});
+
 					const label = createItemsetItemLabel(context, itemset, value);
 
+					const geometry = createComputedExpression(context, itemset.geometry, {
+						defaultValue: '',
+					});
+
+					const nodeElements = itemNode
```

Comment:
The itemNode already includes everything from the `choice` sheet. Here, we follow the same approach as with "value" (L123).